packages/protocol/contracts/adminnable/Whitelister.sol

```diff
@@ -1,115 +1,45 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.6;
 
-import "./RankedAdminnable.sol";
-import "./interfaces/IWhitelister.sol";
-
-/// @title Contract where users are whitelisted until an expiration time or
-/// indefinitely (until the whitelisting is revoked)
-contract Whitelister is RankedAdminnable, IWhitelister {
-    enum AdminRank {
-        Unauthorized,
-        Admin,
-        SuperAdmin
-    }
-
+/// @title Contract where users are whitelisted for specific services (could be
+/// Airnodes, dAPIs, beacons, etc.) until an expiration time or indefinitely
+contract Whitelister {
```

Comment:
I don't see the point of this file anymore. I think we should probably move all of the functionality to RequesterRrpAuthorizer and rename that to something more apt.

Reply:
I think `RrpBeaconServer` is using it, isn't it? 🤔

Reply:
Ah right, then this is fine. The only issue I have with the whitelister is that the setting functions are separated and duplicated across contracts as you've mentioned before. The old abstract way was a cleaner approach, atleast to me.

Reply:
The problem with the old approach were mainly the events. We want to emit `airnode`+`endpointId` and not `serviceId` for the authorizers and this is pretty much the only way of doing that.